content/en/blog/preact-x.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Preact X, a story of stability
+date: 2024-05-24
+authors:
+  - Jovi De Croock
+---
+
+# Preact X, a story of stability
+
+A lot of you have been waiting for [Preact 11](https://github.com/preactjs/preact/issues/2621), announced in an issue opened
+way back in July 2020, and to be clear I was one of the most excited people for v11.
+When we started thinking about Preact 11 we believed that there was no way to introduce the changes we had in mind
+in Preact X without breaking changes, some of the things we had in mind:
+
+- Using a backing VNode structure to reduce GC, by doing this we'd only use the result of `h()` to update our backing-node.
+- Reconciler performance, adding fast paths for mounting/...
+- Some changes like removing `px` suffixing, `forwardRef` and breaking IE11 support.
+- Keeping ref in props.
+- Addressing event/child diffing bugs.
+
+The above were our goals, when breaking these down we can clearly see that there's a distinction here, only point 3 would
+be a breaking change for our consumers while point 1 would have a huge impact to folks building extensions like Preact Signals,
+Preact devtools, ... This could break a lot of the cool stuff our community is working on to extend Preact.
```

Comment:
```suggestion
These were our initial goals for v11, but upon going down this path, we realised that many of those changes weren't actually breaking changes and could be released directly in v10 in a non-breaking way. Only the third point, removing the `px` suffix and passing `ref` directly in props as well as dropping IE11, fall into the breaking changes category. We went with releasing the other features in the stable v10 release line, which allows any Preact user to benefit from them immediately, without having to change their code.

Preact has a much bigger userbase today compared to when we made the original plans for v11. It enjoys wide usage in many small to big companies for mission critical software. We really want to be sure that any breaking changes we may introduce, are absolutely worth the cost of moving the whole ecosystem over to it.
```

Reply:
That last sentence in this suggestion should drop the comma I think, breaks the flow.

Reply:
Good point, updated.